docs/sources/flow/setup/install/windows.md

```diff
@@ -58,6 +58,13 @@ To do a silent install of Grafana Agent on Windows, perform the following steps.
 * `/DISABLEREPORTING=<yes|no>` Disable [data collection][]. Default: `no`
 * `/DISABLEPROFILING=<yes|no>` Disable profiling endpoint. Default: `no`
 
+## Service Configuration
+
+Grafana Agent is using the Windows Registry `HKLM\Software\Grafana\Grafana Agent Flow` for service configuration.
+
+* `Arguments` (Type `REG_MULTI_SZ`) Each value represents a binary argument for grafana-agent-flow binary.
```

Comment:
Do we need any sort of documentation around escaping the values here? Thinking quotes ,double quotes etc

Reply:
I think no. `Arguments` is a multi string value which is natively passed to an go slice. 

Of couse you have to quote think, if you are running reg.exe from CLI. But if you using RegEdit, then no additional quoting is required.

It's could be important to mention that there is no shell parsing. If you quote path, like `"C:\file.json"`, then the Agent complains that the path `"C:\file.json"` does not exists (because the quotes are not stripped)